problems/loops.js

```diff
@@ -162,7 +264,13 @@ function rangeSum() {}
  * @param {number} max number
  * @returns {number[]} all odd numbers in range decreasing
  */
-function rangeOdd() {}
+const rangeOdd = (min, max) => {
+  for (let i = max; i >= min; i--) {
+    if (i % 2) {
+      return i;
```

Comment:
careful of returning in loops. Instead you'll need an output array that you can fill with numbers.